examples/style-helpers/icon-style.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
```

Comment:
Please adapt this example to the template, to have the airship Panel for the title & description